gtsfm/averaging/translation/dummy_ta.py

```diff
@@ -2,33 +2,33 @@
 
 Authors: Ayush Baid
 """
-import abc
```

Comment:
why not just delete the file?

Reply:
we might need it for bypassing averaging though

Reply:
I'll leave it for now then